app/controllers/api/v1/items_controller.rb

```diff
@@ -7,4 +7,19 @@ def index
   def show
     render json: ItemSerializer.new(Item.find(params[:id]))
   end
+
+  def create
+    new_item = Item.new(item_params)
+    if new_item.save
+      render json: ItemSerializer.new(new_item), status: :created
+    else
+      render json: { error: "Missing attribute(s) for item creation" }, status: :unprocessable_entity
```

Comment:
Oooo I wasn't familiar with this way of rendering an error - nice!

I am curious if there's a way to refactor so we can pull the error message from the exception itself (like we did in class today) rather than hard coding it - but only if we have time!